test-util/config.ts

```diff
@@ -64,7 +61,14 @@ export async function config(props?: NodeProps): Promise<Config> {
         break;
       }
     }
-    return new Config(port, process.close.bind(process), props?.altRuntime);
+    return new Config(
+      port,
+      () => {
+        process.kill("SIGKILL");
+        process.close();
```

Comment:
`.close` releases Deno’s memory associated with the process but it doesn't `.kill` the process